homeassistant/components/toon/__init__.py

```diff
@@ -94,6 +196,23 @@ def icon(self) -> str:
         """Return the mdi icon of the entity."""
         return self._icon
 
+    @property
+    def should_poll(self) -> bool:
+        """Return the polling requirement of the entity."""
+        return False
+
+    async def async_added_to_hass(self) -> None:
+        """Connect to dispatcher listening for entity data notifications."""
+        async_dispatcher_connect(
```

Comment:
Store the result of this function call in `self._unsub_dispatcher` and call that result inside `async_will_remove_from_hass`: `self._unsub_dispatcher()`

Reply:
Done.